github-actions/feature-request/action.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Manage Feature Requests'
+description: 'Manages feature requests based on community demand'
+author: 'Angular'
+inputs:
+  token:
+    description: 'A token for the feature request bot.'
+    default: ${{ github.token }}
+  minimum-unique-comment-authors-for-consideration:
+    description: 'Number of unique comment authors required to move a feature request to our consideration list.'
+    default: 10
+  minimum-votes-for-consideration:
+    description: 'Number of votes required to move a feature request to our consideration list.'
+    default: 20
+  feature-request-label:
+    description: 'Label used to distinguish feature request from other issues.'
+    default: 'feature'
+  in-backlog-label:
+    description: 'Label used to distinguish feature requests, which are already part of the backlog.'
+    default: 'in-backlog'
+  requires-votes-label:
+    description: 'Label used to distinguish requests in voting phase from other issues.'
+    default: 'votes-required'
```

Comment:
We don't have a consistent way of naming labels in FW repo, but labels created recently don't have hyphens between words. I'd propose removing hyphens from new labels as well :) (so this label would be `votes required`).